transcription/transcription.go

```diff
@@ -76,3 +77,17 @@ func fileNameFromURL(url string) string {
 	fileName := tokens[len(tokens)-1]
 	return fileName
 }
+
+// Task returns a task function for transcription using transcription functions.
+func Task(audioURL string, emailAddresses []string) func() error {
```

Comment:
Yeah, we may use mocking. The issue is that it's not possible to mock functions of the same package (right now) so we can't directly mock `DownloadFileFromUrl`. We would need to mock `http.Get` to test this so that when we call `DownloadFileFromUrl` nothing actually happens.

Can you add that we need to test this function to the issue you created?